integration_tests/benchmarks/data/load_coco.py

```diff
@@ -0,0 +1,584 @@
+import json
```

Comment:
nit: I'd probably prefer these files to be moved to gists since a) it's not likely we'll need this code again and b) it's a bit messy at times (docstrings, extra comments, etc.). I'm ok with leaving as-is

Reply:
Replaced the dataloader for the `coco-yolo` notebook with this. Now we can pull from cached data when running it.

Integration tests no longer have any data loading or formatting function.